bears/js/JSHintBear.py

```diff
@@ -16,19 +31,377 @@ class JSHintBear(LocalBear, Lint):
 
-        :param jshint_config: The location of the jshintrc config file.
+        :param prohibit_bitwise:                This option prohibits the use of
+                                                bitwise operators.
+        :param prohibit_prototype_overwrite:    This options prohibits
+                                                overwriting.
+                                                prototypes of native objects
+                                                such as ``Array``.
+        :param force_braces:                    This option requires you to
+                                                always put curly braces around.
+                                                blocks in loops and conditionals
+        :param prohibit_type_coercion:          This options prohibits the use
+                                                of ``==`` and ``!=`` in favor
+                                                of ``===`` and ``!==``.
+        :param future_hostile:                  This option enables warnings
+                                                about the use of identifiers
+                                                which are defined in future
+                                                versions of JavaScript.
+        :param prohibit_typeof:                 This option suppresses warnings
+                                                about invalid ``typeof``
+                                                operator values.
+        :param es3:                             This option tells JSHint that
+                                                your code needs to adhere to
+                                                ECMAScript 3 specification.
+        :param force_filter_forin:              This option requires all
+                                                ``for in`` loops to filter
+                                                object's items.
+        :param iterator:                        This option suppresses warnings
+                                                about the ``__iterator__``
+                                                property.
+        :param allow_funcscope:                 This option suppresses warnings
+                                                about declaring variables inside
+                                                of control structures while
+                                                accessing them later from
+                                                outside.
+        :param prohibit_arg:                    This option prohibits the use of
+                                                ``arguments.caller`` and
+                                                ``arguments.callee``.
+        :param prohibit_comma:                  This option prohibits the use of
+                                                the comma operator.
+        :param prohibit_non_breaking_whitespace: This option warns about "non-
```

Comment:
Ahhh found, but requiring from you to realign your documentation would be absolutely mean from me, so keep it :)